packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -466,9 +466,16 @@ const ChallengeTeamWizard: React.FunctionComponent<
     onClose()
   }
 
-  const onConfirmHandlerMap: Record<string, () => Promise<void>> | void = {
+  const onConfirmHandlerMap: Record<
+    string,
+    (() => Promise<void>) | (() => undefined)
```

Comment:
Downstream expects one of these signatures, so I'll just return undefined.

Reply:
You could just have `() => Promise<void>` and `return Promise.resolve()`, if you think that might make it more simple.